config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "shows#index"
-  resources :shows
   resources :users
+  resources :shows do
```

Comment:
Please us the "only" syntax with your routes. I believe you only have actions for index and show at the moment, so only those paths should be available.